kernel/rtlil.cc

```diff
@@ -4044,7 +4059,7 @@ void RTLIL::SigSpec::replace(const RTLIL::SigSpec &pattern, const RTLIL::SigSpec
 			other->bits_[j] = with.bits_[it->second];
 		}
 	}
-	
```

Comment:
Nit: extra newline change

Reply:
Hmmm looks like my editor change an \tab\n to  \n. I can't even seem to put it back. Can we include this tiny cleanup, please?

Reply:
Sure, I think worse have been merged.

Reply:
Thanks :-)